project/src/components/app/room/review-list/review-list.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import ReviewItem from './review-item/review-item.jsx';
+import commentsProp from '../../../prop-types/comments.prop.js';
+
+const NUMBER_OF_REVIEW = 10;
+
+function renderReviewItem(data) {
+  return (
+    <ReviewItem
+      comment = {data.comment}
+      date = {data.date}
+      rating = {data.rating}
+      avatarUrl = {data.user.avatarUrl}
+      name = {data.user.name}
+      key = {data.id}
+    />
```

Comment:
```suggestion
function renderReviewItem({id, ...rest}) {
  return (
    <ReviewItem
      key={id}
      id={id}
      {...rest}
    />
```
и что за пробелы вокруг присвоения?